extensions/positron-assistant/src/participants.ts

```diff
@@ -398,6 +398,11 @@ abstract class PositronAssistantParticipant implements IPositronAssistantPartici
 				xml.node('plots', 'A plot is visible.')
 			);
 		}
+		if (positronContext.currentDate) {
+			positronContextPrompts.push(
+				xml.node('date', `Today's date is: ${positronContext.currentDate}`),
```

Comment:
Thanks, this is better than having a tool do it!

Reply:
I can add the Positron version here too, while I'm here, if you'd like?

Reply:
That'd be great!

Reply:
Done!